README.md

```diff
@@ -37,5 +37,7 @@ These cheatsheets can also be found in the folder named [module-cheatsheets](htt
 - [Intro to R tidyverse cheatsheet](https://github.com/AlexsLemonade/training-modules/blob/master/module-cheatsheets/intro-to-R-tidyverse-cheatsheet.md)
 - [RNA seq cheatsheet](https://github.com/AlexsLemonade/training-modules/blob/master/module-cheatsheets/RNA-seq-cheatsheet.md)
 - [scRNA-seq cheatsheet](https://github.com/AlexsLemonade/training-modules/blob/master/module-cheatsheets/scRNA-seq-cheatsheet.md)
+- [Advanced scRNA-seq cheatsheet](https://github.com/AlexsLemonade/training-modules/blob/master/module-cheatsheets/scRNA-seq-advanced-cheatsheet.md)
+- [`SCE` and `Seurat` objects cheatsheet](https://github.com/AlexsLemonade/training-modules/blob/master/module-cheatsheets/SCE-Seurat-cheatsheet.md)
```

Comment:
I think maybe something a bit more generic? This would give us space to add in the `scverse` content later.
```suggestion
- [Converting to/from `SCE` objects cheatsheet](https://github.com/AlexsLemonade/training-modules/blob/master/module-cheatsheets/SCE-Seurat-cheatsheet.md)
```